notebooks/samples/scikit-learn/custom_prediction/custom-prediction-routine-scikit-learn.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>nit: should PROJECT_ID = "[&lt;your-project-id&gt;]". The reason is that if you keep "&lt;your-project-id&gt;". It will actually set project id to that value, with brackets is able to detect invalid project_id. Also shell script expect the []</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>